Types module not found causing all builds to fail

**UPDATE:** We got it to work by using version 1.90.0 for now. But would like to upgrade to the latest one or may be anything near 1.100.0(the latest at the time of writing is 1.104.0)


Hey,

So today morning all our angular app builds using posthog-js version 1.87.0 started breaking. Below is the error that appears on the workflow runs

`
Error: node_modules/posthog-js/dist/module.d.ts:1:62 - error TS2307: Cannot find module 'types' or its corresponding type declarations.
import { MinimalHTTPResponse as MinimalHTTPResponse$1 } from 'types';
`

I have tried the latest version 1.104.4 too.

Has some breaking change been introduced? I can see that a new release went out 18hrs ago.

--- a/src/autocapture-utils.ts
+++ b/src/autocapture-utils.ts
@@ -1,4 +1,4 @@
-import { AutocaptureConfig, Properties } from 'types'
+import { AutocaptureConfig, Properties } from './types'
 import { _each, _entries, _includes, _trim } from './utils'
 
 import { _isArray, _isNull, _isString, _isUndefined } from './utils/type-utils'

--- a/src/extensions/surveys.tsx
+++ b/src/extensions/surveys.tsx
@@ -1,4 +1,4 @@
-import { PostHog } from 'posthog-core'
+import { PostHog } from '../posthog-core'
 import { Survey, SurveyType } from '../posthog-surveys-types'
 import { SurveysWidget } from './surveys-widget'
 

--- a/src/extensions/surveys/surveys-utils.ts
+++ b/src/extensions/surveys/surveys-utils.ts
@@ -1,4 +1,4 @@
-import { PostHog } from 'posthog-core'
+import { PostHog } from '../../posthog-core'
 import {
     BasicSurveyQuestion,
     LinkSurveyQuestion,
@@ -68,7 +68,7 @@ export const style = (id: string, appearance: SurveyAppearance | null) => {
               padding-top: 10px;
               border-radius: 6px;
               border-color: ${appearance?.borderColor || '#c9c6c6'};
-              margin-top: 14px; 
+              margin-top: 14px;
           }
           .form-submit {
               box-sizing: border-box;

--- a/src/rate-limiter.ts
+++ b/src/rate-limiter.ts
@@ -1,4 +1,4 @@
-import { MinimalHTTPResponse } from 'types'
+import { MinimalHTTPResponse } from './types'
 import { logger } from './utils/logger'
 
 const oneMinuteInMilliseconds = 60 * 1000

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,6 +1,5 @@
 {
     "compilerOptions": {
-        "baseUrl": "./src",
         "outDir": "./lib",
         "target": "ES5",
         "lib": ["dom", "dom.iterable", "esnext"],

